finalPrep/README.md

```diff
@@ -45,30 +45,36 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
+<img width="1200" alt="Screen Shot 2022-02-10 at 7 03 21 PM" src="https://user-images.githubusercontent.com/93043035/153527462-547f3c3d-470a-49d6-a49c-ad607f51b20c.png">
```

Comment:
This looks awesome - I appreciate the family time as well as the hours devoted to Turing - it looks like you are set up for success in terms of managing the time expectations!